RCTAppleHealthKit/RCTAppleHealthKit+Methods_Results.m

```diff
@@ -85,17 +81,17 @@ - (void)results_saveBloodGlucoseSample:(NSDictionary *)input callback:(RCTRespon
 {
     HKQuantityType *bloodGlucoseType = [HKQuantityType quantityTypeForIdentifier:HKQuantityTypeIdentifierBloodGlucose];
 
-    HKUnit *mmoLPerL = [[HKUnit moleUnitWithMetricPrefix:HKMetricPrefixMilli molarMass:HKUnitMolarMassBloodGlucose] unitDividedByUnit:[HKUnit literUnit]];
-
     HKUnit *mgPerdL = [[HKUnit gramUnitWithMetricPrefix:HKMetricPrefixMilli] unitDividedByUnit:[HKUnit literUnitWithMetricPrefix:HKMetricPrefixDeci]];
 
     double value = [RCTAppleHealthKit doubleValueFromOptions:input];
     NSDate *sampleDate = [RCTAppleHealthKit dateFromOptions:input key:@"date" withDefault:[NSDate date]];
+    NSDate *startDate = [RCTAppleHealthKit dateFromOptions:input key:@"startDate" withDefault:sampleDate];
```

Comment:
The documentation should be updated for these dates

Reply:
> The documentation should be updated for these dates

The documentation already has `startDate` and `endDate` and does **not** have `date` for a blood glucose sample. Also, Typescript static type checking for `saveBloodGlucoseSample()` supports this.

> Also, can you please explain to me why there is a `startDate` and `endDate`, as when you save a blood glucose sample there is only a required a date

1. Apple HealthKit implements separate start date and end date for blood glucose samples (even though they can be the same). I'm guess they are not making assumptions on whether it should be a single point.
2. Static types already support using separate dates.

Also note that the changes are backwards compatible, since `startDate` and `endDate` use `date` as the default value.

Reply:
@diatche thanks for the clarification, but I don't know if I'm getting this wrong, but the file `saveBloodGlucoseSample.md` is missing `endDate` and the new field `date`, there is only `startDate` at line 12

Reply:
@Macelai I missed `saveBloodGlucoseSample.md` and only looked at `getBloodGlucoseSamples.md`. The docs for `saveBloodGlucoseSample` we're a bit outdated and some of it was incorrect (you cannot measure blood glucose levels in percentages). Pushed an update.

Considering that the unit was previous ignored in both glucose methods, this would be a good time to fix the unit to the SI unit for blood glucose: mmol/L. See pushed commit.